commands/load.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euo pipefail
+
+arg=""
+
+if plugin_read_value_into_result BUILDKITE_PLUGIN_DOCKER_LOAD ; then
+  arg="${result[0]}"
+fi
+
+if [[ -z "$arg" ]]; then
+  echo "Error: load has to be given a path to the file that should be loaded"
+  exit 1
+fi
+
+echo "load arg: $arg"
```

Comment:
I'm not sure this is necessary... but if you want to print out some message, I would make it more user friendly:

```suggestion
echo "Loading image from $arg"
```